convert/pom.xml

```diff
@@ -21,70 +21,9 @@
         <!--markdown libraries-->
         <dependency>
             <groupId>com.vladsch.flexmark</groupId>
-            <artifactId>flexmark</artifactId>
+            <artifactId>flexmark-all</artifactId>
```

Comment:
this is a bad idea. need to be reverted. text2confl should depend only on features that it uses